packages/table/src/Table.tsx

```diff
@@ -14,6 +20,42 @@ const tableStyles = css`
   border-bottom: 1px solid ${uiColors.gray.light2};
 `;
 
+const ScrollState = {
+  None: 'none',
+  Left: 'left',
+  Right: 'right',
+  Both: 'both',
+} as const;
+
+type ScrollState = typeof ScrollState[keyof typeof ScrollState];
+
+function getScrollShadowStyle(scrollState: ScrollState): string {
+  const shadowColor = transparentize(0.6, uiColors.gray.dark1);
```

Comment:
This should be transparent black, no?

Reply:
I style QA'd this with Rob, but happy to update if you'd like

Reply:
Easy to miss this in a design QA. In general, we've been moving all of our shadows to RGBA values of our black.

Reply:
To be fair, I think the Table Sketch library needs this update as well. @audroue Could you wrap that into the migration to Figma?